modules/ipi-install-network-requirements.adoc

```diff
@@ -40,18 +40,35 @@ For example:
 test-cluster.example.com
 ----
 
+ifdef::upstream[]
+For assistance in configuring the DNS server, check xref:ipi-install-upstream-appendix[Appendix] section for:
```

Comment:
No xrefs allowed in modules. If this is just for upstream, maybe we can remove it or comment it out?

Reply:
Effectively, it is commented out, because there is no upstream definition in our files. It's there so that the upstream and downstream modules are identical, but the xref wont be published so it won't break anything. It just makes it easier for us to synchronize.